builtin/aws/alb/releaser.go

```diff
@@ -304,9 +315,145 @@ func (r *Releaser) Release(
 	return &Release{
 		Url:             "http://" + hostname,
 		LoadBalancerArn: *lb.LoadBalancerArn,
+		TargetGroupArn:  target.Arn,
+		Region:          target.Region,
```

Comment:
I needed the region to build the right client. [Other plugins](https://github.com/hashicorp/waypoint/blob/main/builtin/aws/ec2/platform.go#L95) have this on their config. Not sure which pattern we prefer.